dev/ad55dbd9-494f-4597-a1d4-c5f3157515ca/device.lp

```diff
@@ -0,0 +1,27 @@
+(librepcb_device ad55dbd9-494f-4597-a1d4-c5f3157515ca
+ (name "74XX125")
```

Comment:
I'm not an expert regarding 74xx devices, but probably the name "74XX125" is too generic, isn't it? The device name must make sure it is a quad tri-state buffer in the DIP-14 package. Is there even a consistent naming scheme for 74xx parts across all manufacturers? If not, I think we need to create separate devices for each manufacturer (in the corresponding libraries). Then you would need to remove this device from this pull request so we can at least merge the symbol and the component (which is the most important thing anyway, creating more devices is very easy if symbol, component and package already exist).

Reply:
74XX125 should be sufficiently specific IMO, see https://en.wikipedia.org/wiki/7400-series_integrated_circuits#Part_numbering and https://en.wikipedia.org/wiki/7400-series_integrated_circuits#Families